### ISSUE TITLE
Implement telegram bot

### ISSUE THREAD

**Issue by amaliaibarra** [josejorgers/to-the-rescue#20]
It would be nice to have a bot that automatically posts the alerts on a telegram channel

**Comment by Siddharth1002** [comment]
Can I work on this issue?

**Comment by josejorgers** [comment]
Yes @Siddharth1002! It's all yours.... Thanks for collaborating with this projects

**Comment by Siddharth1002** [comment]
I am working on this issue. Will need some more time.

**Comment by josejorgers** [comment]
There's no rush! Thanks!
